docs/docs-beta/docs/guides/external-systems/apis.md

```diff
@@ -1,4 +1,80 @@
 ---
 title: Connecting to APIs
 sidebar_position: 20
----
\ No newline at end of file
+---
+
+When building a data pipeline, you'll likely need to connect to several external APIs, each with its own specific configuration and behavior. This guide demonstrates how to standardize your API connections and customize their configuration using Dagster resources.
```

Comment:
@jamiedemaria This is great.

@erinkcochran87 @PedramNavid curious what you think in terms of 

I think we should include (abridged) language about when it is good to use resources. E.g:

-------
Accessing an API through a Dagster resource is useful if you want to:

* Parameterize how to access the service at runtime (either through API or UI) via config.
* Surface that configuration in Dagster. 
* Centralize configuration and implementation of this API access. 
* Plug in different implementations of resources in different environments (local dev versus production, for example).

If you don't want any of these features, you should just invoke the external service directly.